packages/core-api/src/apis/implementations/auth/github/GithubAuth.ts

```diff
@@ -87,22 +91,25 @@ class GithubAuth implements OAuthApi, SessionApi {
       },
     });
 
-    const sessionManager = new StaticAuthSessionManager({
+    const sessionManager = new RefreshingAuthSessionManager({
```

Comment:
Not sure about removing the `StaticAuthSessionManager` tbh, I'm thinking we either want to make this client-side implementation support both GitHub Apps and OAuth Apps, or to have a client for each.

I think a first attempt could be whether we can `GithubAuth` instance can have both a `refreshingSessionManager` and a `staticSessionManager`, and then switch which one is being used automatically. It's likely that doesn't work though, and then I'd prolly try making it either an option to the static factory method, or two different static factory methods depending on the use-case.

If we do end up with having to configure the client in the app, then I think we should stick to OAuth by default, and then switch over to GitHub App by default once we build out support for that in the project.